tests/test_web_request.py

```diff
@@ -610,6 +610,26 @@ def test_clone_headers_dict() -> None:
     assert dict(result) == {'a': 'b', 'c': 'd'}
 
 
+async def test_multipart_formdata_file(protocol) -> None:
+    # Make sure file uploads work, even without a content type
+    payload = StreamReader(protocol, loop=asyncio.get_event_loop())
+    payload.feed_data(b"""-----------------------------326931944431359\r
```

Comment:
How about `\r\n` delimiters? I think, this should be tested too.

Reply:
In fact we need `\n` in there. :) Python inserts them automatically when there is a newline:

```
>>> b"""one line\r
... second line"""
b'one line\r\nsecond line'
```

Reply:
this should probably be made more obvious by putting them there explicitly:
```python
payload.feed_data(
    b"adf\r\n"
    b"afds\r\n"
    b"afds\r\n"
)
```

Reply:
Also because line separators may be different under different envs.